test/connection_SUITE.erl

```diff
@@ -142,6 +144,111 @@ gun_connection_lost(_Config) ->
   [_] = meck:unload(),
   ok.
 
+-spec backoff_sleep_time_grow_and_reset(config()) -> ok.
+backoff_sleep_time_grow_and_reset(_Config) ->
+  ConnectionName = ?FUNCTION_NAME,
+  ok = application:set_env(apns, backoff, 0),
+  ok = application:set_env(apns, backoff_ceiling, 10),
+  {ok, ServerPid}  = apns:connect(cert, ConnectionName),
+
+  % 1st reconnect is without delay
+  GunPid = apns_connection:gun_pid(ConnectionName),
+  Timestamp0 = erlang:monotonic_time(millisecond),
+  receive
+    {reconnecting, _} ->
+      Delta0 = erlang:monotonic_time(millisecond) - Timestamp0,
+      true = Delta0 < 1000
```

Comment:
I prefer using `?assert` for nicer error messages, but it is up to you.

Reply:
I was following the style of other assertions